Parallel.node.ts

```diff
@@ -1086,8 +1121,119 @@ async function parallelChatApiRequest(
 		);
 
 		return result;
-	} catch (error) {
-		throw new NodeOperationError(executeFunctions.getNode(), error as JsonObject);
+	} catch (error: any) {
+		// Extract error details from API response
```

Comment:
I'd be curious, is `errorDetail` always found, is it sometimes found, or is it never found? because this code can likely be made much shorter if we understand the shape of error better.